indico/modules/events/management/forms.py

```diff
@@ -168,6 +168,13 @@ def validate_end_dt(self, field):
 
 class EventLocationForm(IndicoForm):
     location_data = IndicoLocationField(_('Location'), allow_location_inheritance=False)
+    own_map_url = URLField(_('Map URL'))
```

Comment:
don't you want to use `wtforms.validators.URL` validator to make sure it is a valid URL?

Reply:
I think everyone (including me) usually assumes that `URLField` would do this by default.. so good point ;x

Reply:
Yep, surprised about this. Adding the URL validator ;)